### PR TITLE
Remove github.com/pkg/errors

### PR DESCRIPTION
**Pull request by kangaechu**
https://github.com/pkg/errors is now archived and it won't be updated.
So, move github.com/pkg/errors to errors.

### CODE REVIEW

**Comment by minamijoyo** [comment]
Thank you for your fix, but the fix looks inconsistent, I mean that some parts doesn't wrap an original error and loses the cause, and some parts wrap it with `%w`. In addition, the standard `errors` package in Go 1.13+ can wrap errors, but it doesn't contain stacktrace.

How about replacing `github.com/pkg/errors` with `golang.org/x/xerrors` instead of the standard `errors` package? The `xerrors` contains stacktrace.

**Comment by kangaechu** [comment]
@minamijoyo 
Thank you for reviewing.
Use `golang.org/x/xerrors` is a good solution, so I rewrite using `golang.org/x/xerrors`.

File: cyqldog/db.go [hunk 1]
```diff
@@ -94,7 +93,7 @@ func buildRecord(row []interface{}, cols []string) (Record, error) {
 	for i, c := range row {
 		s, err := convertToString(c)
 		if err != nil {
-			return record, errors.Wrapf(err, "failt to convertToString: col = %s, type = %T(%v)", cols[i], c, c)
+			return record, xerrors.Errorf("faied to convertToString: col = %s, type = %T(%v): %w", cols[i], c, c, err)
```
Comment (minamijoyo) [inline cyqldog/db.go:96]: Both the original and the fixed code are typos :)

**Comment by minamijoyo** [comment]
@kangaechu Looks almost ok, except for a typo.

File: cyqldog/db.go (hunk 1 above)
Comment (kangaechu) [reply cyqldog/db.go]: ohh...
Fixed typo in cc4fc81

**Review by minamijoyo — APPROVED** [review]
LGTM 👍

**Comment by kangaechu** [comment]
Thank you!

**Pull request merged**